code/game/gamemodes/marker/gamemodes.dm

```diff
@@ -123,4 +117,20 @@ GLOBAL_DATUM_INIT(shipsystem, /datum/ship_subsystems, new)
 	if (GM.marker_active)
 		to_chat(src, "The marker is already active")
 		return
-	GM.activate_marker()
\ No newline at end of file
+	GM.activate_marker()
+
+/**
+
+There is probably a better way to do this, but I've added some more stringent checks to the gamemode win conditions to prevent things like:
+Admin characters being counted as crew, and on their deletion, ending the game
+Non-critical characters like any ghost-roles you may wish to add, or even antags, from counting as "dead crew" (Antags fallback to their own "are the antags dead checks"
+
+*/
+
+//Method to be called when a human player dies for any reason, this will check the amount of players that should be alive, and sees if they're dead or not. We're doing it via an almost signal (bay lacks signals) to avoid processing overhead.
+/datum/game_mode/marker/check_finished()
+	if(!marker_active)
+		return ..()  //Change this to FALSE if you want to override all other win conditions until the marker activates.
+	if(dead_players.len >= round(player_count-(player_count*((minimum_alive_percentage > 0) ? minimum_alive_percentage/100 : 0)))) //Allows the necromorphs a bit of grace so they don't have to hunt down _everyone_, as that could lead to cheese. The ? statement is there to handle a config value of 0% grace, if you decide to go for that.
```

Comment:
one more thing, put a minimum on this, like requiring at least 5 crew total over the round, otherwise it will interfere with debugging 

and maybe neaten up this line a bit